yarn.lock

```diff
@@ -0,0 +1,15453 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we've migrated this project over to https://pnpm.io instead of yarn. the yarn lockfile should not be added to the repository anymore to avoid confusion.

Reply:
Got it, reverted.

Reply:
I ran the upgrade in a seperate PR: https://github.com/simplabs/qunit-dom/pull/1384